app/controllers/sites_controller.rb

```diff
@@ -106,6 +106,7 @@ def users
     return unless authorize_resource(@site, READ_SITE)
 
     @users = ComputedPolicy.authorized_users(Policy::Actions::READ_SITE, @site)
```

Comment:
This was an initial approach that was only necessary because we didn't had roles at that time. We should change this to the list of users who belongs to a role for this site.